WordPress/gutenberg
# [RNMobile] [WIP] First stab at triggering a native action from editor Help

## Description

Trigger the main WP app to create a new Customer Support ticket. Also, offer to jump to the more general support screen, where more options are available (like, Visit the FAQs).

### Related PRs

* https://github.com/wordpress-mobile/gutenberg-mobile/pull/3960

## How has this been tested?

### Test 1
1. Using the WP app and while inside the ediror, hit the 3-dot menu button
2. Tap on the "Help" option
3. The full-screen editor help center will pop up and hit the "Contact Support" button
4. Notice the "Contact Support" chat window emerge:
<img src="https://user-images.githubusercontent.com/1032913/133801564-2d4c836b-7670-49d3-b829-b7b96a7ca32e.png" width="300"/>

### Test 2
1. Using the WP app and while inside the ediror, hit the 3-dot menu button
2. Tap on the "Help" option
3. The full-screen editor help center will pop up and hit the "All Support Options" button
4. Notice the main app help options screen appear:
<img src="https://user-images.githubusercontent.com/1032913/133801719-f8f67b38-628f-4570-825e-eab9c1b9c53a.png" width="300"/>





## Types of changes
1. Adds a new component [HelpGetSupportButton](https://github.com/WordPress/gutenberg/pull/34811/files#diff-b701d34f30b835c40a2f53d0be91e3be56cc8031255846ab9b4e07f8cced5a6bR16) to define a nice styled button for the help pointers
2. Triggers [creating a new Support ticket](https://github.com/WordPress/gutenberg/pull/34811/files#diff-bf50e5ac92a94cf2afdd97fcdcb1295ba8fec9cfd1bc1fdc151279999f4460f0R159) when the "Contact Support" button is tapped
3. Triggers [navigating to the main app Help options screen](https://github.com/WordPress/gutenberg/pull/34811/files#diff-bf50e5ac92a94cf2afdd97fcdcb1295ba8fec9cfd1bc1fdc151279999f4460f0R169) when the "All Support Options" button is tapped
4. Flows above are supported by accompanying bridge methods






## Checklist:
